argilla/docs/how_to_guides/use_markdown_to_format_rich_content.md

```diff
@@ -0,0 +1,177 @@
+
+!!! tip
+    You can get pretty creative with HTML. Think for example about visualizing graphs and tables. Some interesting Python package examples that allow for this are the `pandas.DataFrame.to_html` and the `plotly.io.to_html` methods.
+
+## Multi-modal support: images, audio, video, PDFs and more
+
+Argilla has basic image support in different ways that both have their pros and cons but offer the same UI experience because they both rely on HTML.
+
+![media](../assets/images/how_to_guides/markdown/media.png)
+
+### Local content through DataURLs
+
+A DataURL is a scheme that allows data to be encoded into a base64-encoded string, and then embedded directly into HTML. To facilitate this, we offer some functions: `image_to_html`, `audio_to_html`, `video_to_thml` and `pdf_to_html`. These functions accept either the file path or the file's byte data and return the corresponding HTMurl to render the media file within the Argilla user interface. Additionally, you can also set the `width` and `height` in pixel or percentage for video and image (defaults to the original dimensions) and the autoplay and loop attributes to True for audio and video (defaults to False).
+
+!!! warning
+    DataURLs increase the memory usage of the original filesize. Additionally, different browsers enforce different size limitations for rendering DataURLs which might block the visualization experience per user.
+
+=== "Image"
+
+    ```python
+    from argilla.markdown import image_to_html
+
+    html = image_to_html(
+        "local_image_file.png",
+        width="300px",
+        height="300px"
+    )
+
+    rg.Record(
+        fields={"markdown_enabled_field": html}
+    )
+    ```
+
+=== "Audio"
+
+    ```python
+    from argilla.markdown import audio_to_html
+
+    html = audio_to_html(
+        "local_audio_file.mp3",
+        width="300px",
+        height="300px",
+        autoplay=True,
+        loop=True
+    )
+
+    rg.Record(
+        fields={"markdown_enabled_field": html}
+    )
+    ```
+
+=== "Video"
+
+    ```python
+    from argilla.markdown import video_to_thml
+
+    html = video_to_html(
+        "my_video.mp4",
+        width="300px",
+        height="300px",
+        autoplay=True,
+        loop=True
+    )
+
+    rg.Record(
+        fields={"markdown_enabled_field": html}
+    )
+    ```
+
+=== "PDF"
+
+    ```python
+    from argilla.markdown import pdf_to_html
+
+    html = pdf_to_html(
+        "local_pdf.pdf",
+        width="300px",
+        height="300px"
+    )
+
+    rg.Record(
+        fields={"markdown_enabled_field": html}
+    )
+    ```
+
+### Hosted content
+
+Instead of uploading local files through DataURLs we can also visualize URLs that link directly to media files such as images, audio, video, and PDFs hosted on a public or private server. In this case, you can use basic HTML to visualize content that is available on something like Google Photos or decide to configure a private media server.
```

Comment:
```suggestion
Instead of uploading local files through DataURLs, we can also visualize URLs directly linking to media files such as images, audio, video, and PDFs hosted on a public or private server. In this case, you can use basic HTML to visualize content available on platforms like Google Drive or decide to configure a private media server.
```